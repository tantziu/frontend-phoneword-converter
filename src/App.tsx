import React from 'react';
import {BrowserRouter as Router, Switch, Route,} from 'react-router-dom'
import Home from './pages/Home'
import './App.css';

const App = () => {
  return (
    <Router>
      <div className="App">
      <Switch>
        <Route exact path="/" render={()=> (
          <React.Fragment>
            <Home/>
          </React.Fragment>
        )}/>
      </Switch>
      </div>
    </Router>
  );
}

export default App;
