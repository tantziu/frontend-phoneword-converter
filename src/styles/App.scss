@import "color";

.App {
    margin: 0;
    color: $color-black;
    font: 400 18px CoreSans, Arial, sans-serif;
}
