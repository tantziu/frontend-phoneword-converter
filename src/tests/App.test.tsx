import React from 'react';
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import { render, screen } from '@testing-library/react'
import App from '../App'

// test('renders learn react link', () => {
//   render(<App />)
//   const linkElement = screen.getByText(/learn react/i)
//   expect(linkElement).toBeInTheDocument()
// })

describe('App', () => {
  it('render without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<App />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('has a valid snapshot', () => {
    const component = renderer.create(<App/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
