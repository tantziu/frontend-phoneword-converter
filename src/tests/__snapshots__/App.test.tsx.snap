// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App has a valid snapshot 1`] = `
<div
  className="App"
>
  <div
    className="HomePage"
  >
    <div
      className="Search"
    >
      <h3>
        Input:
      </h3>
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
      <button
        className="button"
        onClick={[Function]}
        type="button"
      >
        Convert
      </button>
    </div>
    <div
      className="table"
    >
      <h3>
        Words:
      </h3>
      <ul />
    </div>
  </div>
</div>
`;
